NQueens.py

```diff
@@ -0,0 +1,57 @@
+# Time Complexity : O(N*N) Required for board and recursive stack.
```

Comment:
Actually its n! bcz, initially you can put queen at any place out of n places, then you can put out of n-2 , then n-4 so approximately it becomes n!